app/views/shared/_sidebar.html.erb

```diff
@@ -0,0 +1,30 @@
+<nav id="sidebar" class="sidebar-full hidden">
+  <div class="sidebar-outside">
+    <div class="absolute inset-0 bg-black opacity-50" tabindex="0"></div>
+  </div>
+  <aside class="sidebar">
+    <span class="sidebar-close">
+    <div class="mr-2 ml-auto mt-2 fill-current w-6 h-6 cursor-pointer">
+      <i class="fas fa-times"></i>
+    </div>
+    </span>
+    <span class="">
+      <%= image_tag 'tribo-logo-light.svg', class: 'sidebar-logo' %>
+    </span>
+    <span class="flex p-4 sidebar-item"><button id="modal-register" class="btn font-bold text-base text-left">Regístrate / Mi cuenta</button></span> 
+    <span class="flex p-4 sidebar-item"><button id="modal-recommendations" class="btn font-bold text-base text-left">#QuédateEnCasa</button></span>
+    <span class="flex p-4 pt-0 text-base text-left">Recomendaciones para el aislamiento preventivo</span>
+    <span class="flex p-4 sidebar-item"><button id="modal-terms" class="btn font-bold text-base text-left">Términos y condiciones</button></span> 
+    <span class="flex p-4 sidebar-item"><button id="modal-faq" class="btn font-bold text-base text-left">Preguntas y respuestas</button></span> 
+    <span class="flex p-4 sidebar-item"><button id="modal-policy" class="btn font-bold text-base text-left">Políticas de Privacidad</button></span>
```

Comment:
The p-4 class used here in every item overwrites the padding inside the sidebar-item, making the elements look  wider than they should look like.
Also, the flex property added here in every instance was deleted from the class sidebar-item, making kinda odd the decision to remove it in the first place.
I suggest to erase these classes here and put them in the generic class.
Also, I did not add margins in the generic class because I saw that you didn't count the recommendations for COVID inside the button, so the behavior of the margins would change. I would recommend to add those margins according to the design (If i remember well its 1.5rem between items and 4.5 between the menus and the sign-in/log-out button